### ISSUE TITLE
Cannot Right Click from Taskbar

### ISSUE THREAD

**Issue by ForeEyes** [ForeEyes/PodcastFeed#28]
Right clicking on taskbar window only opens it, instead of showing a menu to exit...

**Comment by pureooze** [comment]
Yep this should be fixed by https://github.com/ForeEyes/PodcastFeed/pull/27

**Comment by ForeEyes** [comment]
After #27, the close right click menu only shows when the window is visible...

**Comment by ForeEyes** [comment]
also can you talk right now?

**Comment by pureooze** [comment]
Yeah I will go on now
